_components/form-templates.md

```diff
@@ -175,61 +178,12 @@ lead: Patterns for some of the most commonly used forms on government websites.
   </div>
 </div>
 
-<div class="preview">
```

Comment:
It's not in the side nav, but it is between the address form and the login form.